### ISSUE TITLE
with initial water level not clear that you can check the boxes

### ISSUE THREAD

**Issue by Jonasvsl** [nens/threedi-api-qgis-client#70]
* threedi-api-qgis-client version: 1.5.0.b
* Python version:
* Operating System:

I expect them to be white when you can check them, now they're grey on the inside

### Description

Describe what you were trying to get done.
Tell us what happened, what went wrong, and what you expected to happen.

### What I Did

```
Paste the command(s) you ran and the output.
If there was a crash, please include the traceback here.
```

**Comment by Jonasvsl** [comment]
![image](https://user-images.githubusercontent.com/1453159/86112781-b6b8a080-bac8-11ea-9ae4-64bd2cd3ea75.png)

**Comment by DamnBack** [comment]
Fixed in the version 2.0.0
